### PR TITLE
Modify the protect-integrity tool to print result on file

### PR DESCRIPTION
**Pull request by liqinggd**

### CODE REVIEW

**Review by jessehui — COMMENTED** [review]
Generally looks good. Just some small suggestions.

File: tools/occlum_build.mk [hunk 1]
```diff
@@ -60,13 +50,16 @@ $(SIGNED_ENCLAVE): $(LIBOS)
...
 		ln -sf "libocclum-libos.so.$(major_ver)" libocclum-libos.so ; \
-		echo -e "$$OCCLUM_BUILTIN_SYS_CONF_FILE_MAC\c" > temp_mac_file && \
-		objcopy --update-section .builtin_config=temp_mac_file libocclum-libos.so && \
-		rm temp_mac_file
+		LD_LIBRARY_PATH="$(SGX_SDK)/sdk_libs" "$(occlum_dir)/build/bin/occlum-protect-integrity" \
+			show-mac \
+			"$(instance_dir)/build/.Occlum_sys.json.protected" \
+			temp_sys_mac && \
+		objcopy --update-section .builtin_config=temp_sys_mac libocclum-libos.so && \
+		rm -f temp_sys_mac
```
[the remark this answers is not in the document]
Comment (liqinggd) [reply tools/occlum_build.mk]: Good suggestion, thanks for the review

File: tools/protect-integrity/App/App.cpp [hunk 2]
```diff
@@ -271,15 +283,26 @@ static int parse_args(
...
+    char **arg_output_path) {
+    if (argc < 3) { return -1; }
 
     if (strcmp(argv[1], "protect") == 0) {
         *arg_command = CMD_PROTECT;
     } else if (strcmp(argv[1], "show") == 0) {
         *arg_command = CMD_SHOW;
+        if (argc == 4) {
+            *arg_output_path = argv[3];
+        } else if (argc > 4) {
```
[the remark this answers is not in the document]
Comment (liqinggd) [reply tools/protect-integrity/App/App.cpp]: Good suggestion, thanks for the review

File: tools/occlum_build.mk [hunk 3]
```diff
@@ -108,13 +101,16 @@ $(INITFS_IMAGE): $(INITFS) $(INITFS_DIRS) $(INITFS_FILES) $(IMAGE_CONFIG_JSON) $
 		"$(INITFS_IMAGE_MAC)"
 
 $(IMAGE_CONFIG_JSON): $(instance_dir)/build/Occlum.json.protected
-	@export OCCLUM_CONF_FILE_MAC=`$(get_occlum_user_conf_file_mac)` ; \
-		echo "EXPORT => OCCLUM_CONF_FILE_MAC = $$OCCLUM_CONF_FILE_MAC" ; \
+	@LD_LIBRARY_PATH="$(SGX_SDK)/sdk_libs" "$(occlum_dir)/build/bin/occlum-protect-integrity" \
```
[the remark this answers is not in the document]
Comment (liqinggd) [reply tools/occlum_build.mk]: Good suggestion, thanks for the review

**Review by jessehui — APPROVED** [review]
LGTM

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**